el/solid_test.go

```diff
@@ -0,0 +1,21 @@
+package el_test
+
+import (
+	g "github.com/maragudk/gomponents"
+	"github.com/maragudk/gomponents/assert"
+	"github.com/maragudk/gomponents/attr"
+	"github.com/maragudk/gomponents/el"
+	"testing"
```

Comment:
The project's import order is stdlib before internal imports.

Reply:
OK. Will respect this in the future.